### DISCUSSION TITLE
Help with socket communication

### DISCUSSION THREAD

**Discussion by qxdevelop** [AxisCommunications/discussions#165] [General]
Hello, I'm trying to use socket communication with M1137 Axis through postman for testing it but I do not get success in the communication. I'm doing what says in the WebSocket guide of AXIS OS PORTAL. Look the capture

![connection_test](https://user-images.githubusercontent.com/118343592/214889667-7c878706-3978-49d7-b6bf-bb25db1c43ff.jpg)

Using the WebSocket King client as example it works. 
Please, Could you help me? I dont know what i'm doing wrong.   The idea is create a websocket client for python.

**Comment by qxdevelop** [comment]
Also I tried ws://user:password@ip/vapix/wa-data-stream?sources=events with the parameters but it does not work.

**Reply by onie5** [reply]
Hi,

Just to confirm, are you using "GET" method?

JSON data is sent using POST.

**Comment by vivekatoffice** [comment]
Related link:
https://github.com/orgs/AxisCommunications/discussions/263
